go.mod

```diff
@@ -3,11 +3,13 @@ module github.com/odpf/salt
 go 1.16
 
 require (
+	github.com/go-resty/resty/v2 v2.6.0
```

Comment:
Is this module being used? I don't see it in the imports in the PR changes.

Reply:
I was thinking to use it for testing , if not going to use `resty` then will surely remove it

Reply:
👍